server/graphql/v2/mutation/CollectiveMutations.ts

```diff
@@ -47,7 +47,10 @@ const collectiveMutations = {
       }
       const canApply = await host.canApply();
       if (!canApply) {
-        throw new Error('Host is not open to applications');
+        // in the case that the host is the USER, it will be checked in changeHost and addHost whether they are admin of the Collective they are trying to host
+        if (host.type !== 'USER') {
+          throw new Error('Host is not open to applications');
+        }
```

Comment:
I think the proper solution is to delete that whole block.

`addHost` will be smart enough to do the right thing.

Reply:
See:

```
    // If we can't automatically approve the collective and it is not open to new applications, reject it
    if (!shouldAutomaticallyApprove && !hostCollective.canApply()) {
      throw new Error('This host is not open to applications');
    }
```